Dockstore.cwl

```diff
@@ -14,7 +14,7 @@ dct:contributor:
 
 requirements:
 - class: DockerRequirement
-  dockerPull: quay.io/pancancer/pcawg-dkfz-workflow:2.0.8_cwl1.0
+  dockerPull: quay.io/pancancer/pcawg-dkfz-workflow:increase_memory_setting
```

Comment:
Maybe `2.0.9_cwl1.0` or `2.0.8_cwl1.0-increase_memory_setting` to give an indication where this release falls in relation to others?

Reply:
commented on it here: https://github.com/ICGC-TCGA-PanCancer/CGP-Somatic-Docker/pull/18#discussion_r264438904 

The idea is to make it match the branch name in the PR, once PR is merged and ready to be release, the tag will be updated to match the release tag. How does this sound? Any alternative suggestions we'd love to hear as well.

Reply:
sounds good